config/rewrites.d/docs.rackspace.com.json

```diff
@@ -0,0 +1,67 @@
+{
+	"docs.rackspace.com": [
+		{
+			"description": "Redirect root traffic to developer.rs.com/docs/",
+			"from": "^\\/$",
+			"to": "/docs/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 302
+		},
+		{
+			"description": "cloud servers legacy",
+			"from": "^\\/servers/api/v1.0/cs-devguide\\/$",
+			"to": "/docs/#docs-cloud-servers-legacy",
```

Comment:
@kenperkins This "to" URL is just the docs landing page. The hash portion of the URL doesn't correspond to any anchors on that page.